2.4/root/usr/bin/run-mongod

```diff
@@ -74,26 +74,25 @@ if [ -z "${MONGODB_REPLICA_NAME-}" ]; then
     usage
   fi
   # Run the MongoDB in 'standalone' mode
-  if [ ! -f ${MONGODB_DATADIR}/.mongodb_datadir_initialized ]; then
-    # Create MongoDB users and restart MongoDB with authentication enabled
-    # At this time the MongoDB does not accept the incoming connections.
-    mongod $mongo_common_args & #--bind_ip 127.0.0.1 --quiet >/dev/null &
-    wait_for_mongo_up
+  mongod $mongo_common_args &
+  wait_for_mongo_up
+  js_command="db.system.users.count({'user':'admin'})"
+  if [ "$(mongo admin --quiet --eval "$js_command")" == "1" ]; then
```

Comment:
To leave it as is.

If `mongo admin` errors out, it's unlikely that it will output "1". In any case, no matter what branch of the  `if-else` we go, either `mongo_create_admin` or `mongo_reset_admin` will fail and `exit 1`.